python/src/scipp/plot/toolbar.py

```diff
@@ -66,6 +66,19 @@ def __init__(self, canvas=None, ndim=1):
 
         if ndim < 3:
             self.add_button(name="save_view", icon="save", tooltip="Save")
+        else:
+            self.add_button(name="camera_x_normal",
```

Comment:
on left menu, I would say home button and camera setting should be grouped. Other buttons relate to colour scale. At moment those are mixed together.